docker/docs/builder/Dockerfile

```diff
@@ -6,10 +6,11 @@ ARG HTMLTEST_VERSION=0.17.0
 RUN CGO_ENABLED=0 go install github.com/wjdp/htmltest@v${HTMLTEST_VERSION} \
```

Comment:
The fact we build htmltest statically saves us from musl-hell, but I'd suggest as well moving golang to a bookworm flavor

Not a blocker, maybe in the following PRs